mslib/mscolab/conf.py

```diff
@@ -103,6 +103,19 @@ class default_mscolab_settings:
     </FlightTrack>
     """
 
+    XML_CONTENT_INIT = """<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we need this only for tests, or? at least we should only need it there.
We don't expect a user to change this on the server, right?

so we can import it from anywhere, it has currently a strong relation to seed, also define it there. When it is only related to tests it would be tests.constants.